paymentsheet/src/main/java/com/stripe/android/paymentsheet/FormFragment.kt

```diff
@@ -12,7 +12,9 @@ import androidx.fragment.app.Fragment
 import androidx.fragment.app.activityViewModels
 import com.stripe.android.paymentsheet.forms.SofortForm
 import com.stripe.android.paymentsheet.forms.SofortFormViewModel
+import kotlinx.coroutines.ExperimentalCoroutinesApi
 
+@ExperimentalCoroutinesApi
```

Comment:
Any considerations to keep in mind when using this experimental API? When is it expected to become stable?

Reply:
By switching from mapLatest to map I was able to remove the ExperimentalApi annotation.   Because the transform is a very small lightweight operation I don't think we need the mapLatest.